src/App.js

```diff
@@ -400,10 +395,7 @@ export default function App() {
                 alt={'github'}
               />
               Contribute to the tool{' '}
-              <a
-                href="https://github.com/denishamann/arena-stats-tbc-visualizer"
-                target={'_blank'}
-              >
+              <a href="https://github.com/denishamann/arena-stats-tbc-visualizer">
```

Comment:
target blank wasn't needed? is it the default behaviour to open it in a new tab?

Reply:
At first there was no target blank. 
I've added it in a previous PR, then removed it here, because it was triggering a security warning. After some research, it seemed to me like the best practice was... "don't use target blank". But idk, need to dig that a little deeper.